client/src/pages/account/requests/index.js

```diff
@@ -1,14 +1,51 @@
 import React from 'react'
-import { Box, Tabs, Tab } from 'grommet'
+import { Box, Tabs, Tab, Text } from 'grommet'
 import { DrillDownNav } from 'components/DrillDownNav'
 import { AccountEmptyPage } from 'components/AccountEmptyPage'
+import ManageRequestsTable from 'components/ManageRequestsTable'
+import { Loader } from 'components/Loader'
+import { useUser } from 'hooks/useUser'
+import api from 'api'
+import RequestOverviewCard from 'components/RequestOverviewCard'
 import EmptyRequests from '../../../images/empty-requests.svg'
 
 const Requests = () => {
-  const receivedRequests = []
-  const sentRequests = []
+  // fetch the requests
+  const { user, token } = useUser()
+  const [requests, setRequests] = React.useState()
+  const [assigned, setAssigned] = React.useState([])
+  const [received, setReceived] = React.useState([])
+  const [sent, setSent] = React.useState([])
+  const hasReceived = received && received.length > 0
+  const hasSent = sent && sent.length > 0
+  const hasAnyRequests = hasReceived || hasSent
 
-  const hasAnyRequests = receivedRequests.length > 0 || sentRequests.length > 0
+  React.useEffect(() => {
+    const fetchRequests = async () => {
+      const requestsRequest = await api.requests.filter({ limit: 1000 }, token)
```

Comment:
We should consider making the api endpoint filterable. not a concern for now, but soon.

Reply:
Yeah I forget why we moved away from that but we did discuss it.